docs/faq.md

```diff
@@ -93,6 +93,16 @@ If your fuzz target is running for many days and does not find bugs or new cover
 
 In either case, look at the [coverage reports](clusterfuzz.md#coverage-reports) for your target(s) and figure out why some parts of the code are not covered. 
 
+## Why are code coverage reports public?
+
+We work with open source projects and try to keep as much information public as
+possible. The only information that we keep private is the following:
+- Bugs that were found in the past 90 days and were not fixed 30+ days ago.
+- Corpora, as it may contain inputs that are one-mutation-off a bug reproducer.
+
+We believe that making code coverage reports public does not put users at risk,
```

Comment:
Maybe better rephrased as:

We believe that public code coverage reports public do not put users at risk, as they do not indicate the presence of bugs or lack thereof.

Reply:
Thanks, that's certainly better! :)